server/dal/stock-markets.js

```diff
@@ -0,0 +1,98 @@
+const mongoose = require("mongoose");
+require('dotenv').config({path: '../.env'});
+
+mongoose.connect(process.env.STOCK_MARKET_MONGO_DB_CONNECTION_STRING, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+})
+
+var Schema = mongoose.Schema
+
+var stockMarketSchema = new Schema({
+    market_name: String,
+    stocks: Array({
+        name: String,
+        ticker: String
+    })
+}, {collection: 'StockMarket'})
+
+exports.stockMarketInfo = mongoose.model('StockMarket', stockMarketSchema)
+
+
+var realtimeStockInfoSchema = new Schema({
+    market_name: String,
+    market_id: String,
+    stock_name: String,
+    ticker_id: String,
+    stock_id: String,
+    stock_details: {
+        timestamp: Date,
+        previous_close: Number,
```

Comment:
You only used previous_close here which is good, but I think forgot to remove last_price from other areas.